boards/arm/mimxrt1062_fmurt6/mimxrt1062_fmurt6_defconfig

```diff
@@ -0,0 +1,21 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2023 NXP
+#
+
+CONFIG_SOC_MIMXRT1062=y
+CONFIG_SOC_SERIES_IMX_RT=y
+CONFIG_BOARD_MIMXRT1062_FMURT6=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_PINCTRL=y
+CONFIG_FLASH=y
```

Comment:
Why is `CONFIG_FLASH` enabled by default for this platform? Are flash drivers needed for the "minimal" build- IE running hello world on the board?

Reply:
It was added for an experiment, removed it now.